src/core/internal/modes/contract.ts

```diff
@@ -415,9 +415,19 @@ export function contract(parameters: contract.Parameters = {}) {
         const { account, id, internal } = parameters
         const { client } = internal
 
-        const key = account.keys?.find((key) => key.id === id)
+        const key = account.keys?.find(
+          (key) => key.id.toLowerCase() === id.toLowerCase(),
```

Comment:
These should be normalized already from https://github.com/ithacaxyz/porto/pull/408
```suggestion
          (key) => key.id === id,
```